test/e2e/rte/rte_test.go

```diff
@@ -200,14 +200,8 @@ var _ = ginkgo.Describe("with a running cluster with all the components", func()
 				klog.Infof("RTE config: %#v", rc)
 				gomega.Expect(err).ToNot(gomega.HaveOccurred())
 
-				// we intentionally don't check the values themselves - atm this would
-				// be to complex, effectively rewriting most of the controller logic
-				if len(kc.ReservedSystemCPUs) > 0 {
-					gomega.Expect(rc.Resources.ReservedCPUs).ToNot(gomega.BeEmpty())
-				}
-				if len(kc.ReservedMemory) > 0 {
-					gomega.Expect(rc.Resources.ReservedMemory).ToNot(gomega.BeEmpty())
-				}
+				gomega.Expect(rc.TopologyManagerPolicy).To(gomega.Equal(kc.TopologyManagerPolicy), "TopologyManager Policy mismatch")
```

Comment:
I would also add what we go was what expected.

Reply:
the annotation is adding context to the expected annotation, so the value got vs expected should already be there in the failed expectation error message (+ the extra annotations)